src/utils/componentUtilityFunctions.js

```diff
@@ -145,6 +153,14 @@ export const printGrid = grid => {
   console.log(result); // eslint-disable-line
 };
 
+/**
+ *
+ * @param {*} x  the current x location of a card
+ * @param {*} y  the current y location of a card
+ * @param {*} w  current width of a card
+ * @param {*} h  current height of a card
+ * @param {*} grid nested array of rows and columns with the current card index that is occupying them. for example, if the entire top row was taken by the first card it would look like this [[1, 1, 1, 1], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]]
+ */
 export const canFit = (x, y, w, h, grid) => {
   // console.log(`canFit? x=${x}, y=${y}, w=${w}, h=${h}`);
```

Comment:
Remove this? Unless its only being used by tests / storybook. Its also commented out